packages/shared/src/components/icons/ChecklistA/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { ReactElement } from 'react';
+import Icon, { IconProps } from '../../Icon';
+import FilledIcon from './filled.svg';
+import OutlinedIcon from './outlined.svg';
+
+const ChecklistAIcon = ({ size, ...rest }: IconProps): ReactElement => (
```

Comment:
I'm still a bit confused what this means `ChecklistA` can't we give it a more explicit name?
Pretty sure we are not using `Checklist` anyway so maybe just use that?

Reply:
This is how it was in sketch, thought it would be best to just name them the same so down the line we know which icons we added and which ones we are missing.

Reply:
Oh I see, would you mind checking with Design if they can rename it? 😅
If not it's fine, but would clean it a bit.

Reply:
I'll just rename it, don't want to waste time on.